src/local_sensitivity/interpolating_adjoint.jl

```diff
@@ -46,11 +50,11 @@ end
     jac_config = nothing
     uf = nothing
   else
-    uf = DiffEqDiffTools.UJacobianWrapper(f,tspan[2],p)
+    uf = DiffEqDiffTools.UJacobianWrapper(f,tspan[1],p)
     jac_config = build_jac_config(sensealg,uf,u0)
   end
 
-  y = copy(sol(tspan[1])) # TODO: Has to start at interpolation value!
+  y = copy(sol(tspan[1]))
```

Comment:
why does it have to be the interpolation value?

Reply:
is this actually just a copy thing?

Reply:
Should I change it to `y = copy(sol.u[end])`?

Reply:
it's interesting that this is a thing. I don't remember exactly why, or maybe we never did, but it sounds plausible since it mutates `y` in the algorithm. But the end point is always `tspan[1]`, right? So a copy would make more sense.